### ISSUE TITLE
Shared database security

### ISSUE THREAD

**Issue by jcalfee** [dokku/dokku-postgres#324]
Do you have a way to keep the database password hidden from the linked app?  For security, I would like all linked apps to use their own login.  I gather that I can `config:set` each app with its own DATABASE_URL, however, I noticed there are other variables as well that contain the password.  I assume that I can't simply change the password outside of dokku without breaking something (backups for example).

DOKKU_POSTGRES_AQUA_URL
DOKKU_POSTGRES_BLACK_URL
DOKKU_POSTGRES_BLUE_URL

And in the future?  If there is a simple future-proof way to keep the credential separate from the app in the container then the end-user can handle any setup requirements that comes along by falling back to `config:set` and without risking any exposure to the main postgres login.

**Comment by josegonzalez** [comment]
No, there is no way to do this, and I don't intend to implement this. We may at some point implement multiple users/databases within a single datastore service, but thats covered by other systems. Regardless, if the user has access to read configuration, they'll have access to read the database config as well (because its just set as another environment variable).
